bentoml/artifact/keras_model_artifact.py

```diff
@@ -23,7 +23,7 @@
 
 try:
     import tensorflow as tf
-    from tensorflow.python import keras
+    import keras
```

Comment:
This may break for some users who only have installed Tensorflow but not the Keras package, although the original approach has some limitations too, we will do a follow-up PR to address this before next release

Reply:
Should've posted a comment / question on this as well: when actually attempting to pass in my Keras model, `isinstance(data, keras.engine.training.Model)` from [Line 95](https://github.com/ollano-ops/BentoML/blob/38e551a0d6687d2272b2ec454fa9a3af3ecaacc4/bentoml/artifact/keras_model_artifact.py#L95) would return `False`. Switching to this fixed that. It looks as though `tensorflow.python.keras.engine.training.Model` and `keras.engine.training.Model` are not perfect aliases of this. 

I'll create an issue and see if I can put out a fix.